core/provider/service.go

```diff
@@ -209,6 +211,23 @@ func (s *Service) Update(ctx context.Context, p *domain.Provider) error {
 		}
 	}
 
+	go func() {
+		s.logger.Info("fetching resources", "provider_urn", p.URN)
```

Comment:
Can't we have log something like this:
`provider update fetching resources` to debug easier in case of update provider?